<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Soren Harward &lt;stharward@gmail.com&gt;</author>
 <class>TagMatchEditWidget</class>
 <widget class="QWidget" name="TagMatchEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="KDE::DoNotExtract">Form</string>
  </property>
  <property name="whatsThis">
   <string>An editor for a TagMatch constraint.  The user can specify a track characteristic, and the APG will fill the playlist with tracks that match that characteristic.  This functions similarity to the matching attributes of the smart playlists in Amarok 1.4.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Match Tag Constraint Settings</string>
     </property>

       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="MetaQueryWidget" name="attributeQuery"/>
        </item>

        <item row="1" column="0">
         <widget class="QLabel" name="label_Invert">
          <property name="whatsThis">
           <string>The APG will choose all tracks that do not match the specified characteristics.</string>
          </property>
          <property name="text">
           <string>Invert:</string>
          </property>
          <property name="buddy">
           <cstring>checkBox_Invert</cstring>
          </property>
         </widget>
        </item>

        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_Invert">
          <property name="whatsThis">
           <string>The APG will choose all tracks that do not match the specified characteristics.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>

        <item row="2" column="0">
         <widget class="QLabel" name="label_Strictness">
          <property name="whatsThis">
           <string>How strict the APG should be about matching the playlist duration to the specified time.</string>
          </property>
          <property name="text">
           <string>Match:</string>
          </property>
          <property name="buddy">
           <cstring>slider_Strictness</cstring>
          </property>
         </widget>
        </item>

        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_Fuzzy">
            <property name="text">
             <string>fuzzy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Strictness">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>How strict the APG should be about matching the playlist duration to the specified time.</string>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Exact">
            <property name="text">
             <string>exact</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        
        <item row="3" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
        </item>

       </layout>

    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
